pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -20,7 +20,6 @@ import (
 func TestFeatureToggleFiles(t *testing.T) {
 	legacyNames := map[string]bool{
 		"httpclientprovider_azure_auth": true,
-		"service-accounts":              true,
```

Comment:
this needed to be removed to make tests run.
@bergquist 

Unsure if this is necessary anymore?